MVPFragment/root/src/app_package/classes/Fragment.java.ftl

```diff
@@ -1,12 +1,13 @@
 package ${packageName}.view.impl;
 
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+
```

Comment:
I don't think this extra line break is needed

Reply:
I deleted this line, please check.